include/freertos/FreeRTOSConfig.h

```diff
@@ -100,6 +100,7 @@
 /* !!!! configMAX_SYSCALL_INTERRUPT_PRIORITY must not be set to zero !!!!
 See http://www.FreeRTOS.org/RTOS-Cortex-M3-M4.html. */
 #define configMAX_SYSCALL_INTERRUPT_PRIORITY     191 /* equivalent to 0xa0, or priority 5. */
+#define configASSERT( x ) if (!(x)) diewith(BLINK_DIE_ASSERT)
```

Comment:
Do you want this to be in our production releases?
Consider filing a bug (at least in our production collaborators' issue tracker) to make sure this is optimized before release.